src/lib.rs

```diff
@@ -1,6 +1,3 @@
-#![warn(missing_docs)]
-#![doc(html_root_url = "https://docs.rs/sunk/0.1.2")]
-
```

Comment:
> Removed the mandatory documentation enforcement (since it was producing hundreds of redundant errors)

Warnings are meant to be followed-up on, but with the mass exposure of previously-private fields I can see why. The documentation HTML root should remain, though.

Reply:
Sounds good to me! Yeah, I'll re-add that warning and push the allows down. 


Re: `html_root_url` -- I might be wrong, but I think it's a no-op in terms of third-party generated docs. For example, docs.rs (which is what crates.io links to) overrides it ([source](https://github.com/rust-lang/api-guidelines/issues/229#issuecomment-729974673)).

Reply:
TIL. Seems fine to leave it out then.

Reply:
Okay, done -- I've allowed missing docs on a case-by-case basis.